tests/integration_tests/functional/test_aarch64_armshaker.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Opcode fuzzer test for aarch64."""
+
+# pylint: disable=too-many-lines
```

Comment:
nit: the file is not so long to need this.

Reply:
removed